### PR TITLE
PUB-3754 | Fix grid color picker failing to save color change

### PR DESCRIPTION
**Pull request by MayaUribe**
### Description

The color picker in the Shop Grid section started failing at some point and we are unable to change the color.

### Context & Notes
[PUB-3754](https://buffer.atlassian.net/browse/PUB-3754): Fix grid color picker failing to save color change

### Screenshots
https://user-images.githubusercontent.com/988972/135817104-aad1989e-9b1d-4f7e-9cd8-dfab55a144f4.mp4

### Best Practices Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [x] I have added tests to cover my changes.
-   [x] I have tested different user stories.
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

### ⚠️ Pre-deploy Checklist

- [x] I have manually loaded the URL of the staging deployment and confirmed it's working (see below**)
- [x] I have confirmed that the Cypress E2E test is passing (_Note: we have a known issue where it doesn't run on some PRs. Hopefully this will be fixed soon._)
- [x] I have confirmed that all other tests are passing.

** To visit the URL of the staging deployment, please vlick the link in the bot comment, or look under checks at the bottom of this PR and click "Details" next to `bufferbotbrains/cicd-buffer-publish`
image

### CODE REVIEW

File: packages/shared-components/ColorPicker/index.jsx [hunk 1]
```diff
@@ -52,10 +52,6 @@ const ColorPicker = ({ onChange, label, defaultColor, onBlur }) => {
                 onBlur(color, getColorContrast(color));
               }
             }}
-            onBlur={() => {
-              setVisible(false);
-              if (onBlur) onBlur(color, getColorContrast(color));
-            }}
```
Comment (MayaUribe) [inline packages/shared-components/ColorPicker/index.jsx:58]: This blur seems to be closing the popup every time we click on a color.

**Comment by A-Farmer** [comment]
Hey there, @MayaUribe! Would this one be ready to go? 😄

**Comment by MayaUribe** [comment]
> Hey there, @MayaUribe! Would this one be ready to go? 😄

Heya @A-Farmer I think it should be ready! I'll open it for review 😊

**Review by guldenmw — APPROVED** [review]
nice-nooice

**Pull request merged**